### ISSUE TITLE
Problems with Gamepad

### ISSUE THREAD

**Issue by krasnoglaz** [CodeArtemis/TriggerRally#56]
I'm using Xbox 360 gamepad on Windows and on Linux. My right analog stick (throttle) is slightly off center(1500 out of +/- 32k range), and absence of deadzones causes car to be slower noticeably, if I use button or trigger for throttle. Same thing with steering axis when I use D-pad for steering. Other minor problem is that there is no way to restart race from the gamepad without using keyboard.

**Comment by jareiko** [comment]
There is actually a deadzone, but it's currently just 1%. 1.5/32 is about 4.5%, so I increased the deadzone to 5% now. Has it fixed the problem?

Adding a restart race button is an excellent idea. :) It will require a bit more work though.

**Comment by krasnoglaz** [comment]
> Has it fixed the problem?

Yeah. Car doesn't try to drive in opposite direction at the start anymore.
